services/NotificationsService.js

```diff
@@ -8,141 +7,97 @@ Noty.overrideDefaults({
+  const notification = _createNotification(message, type, timeout);
+  notification.show();
 
-  /**
-   * Shows the success notification.
-   *
-   * @static
-   * @param {string} message Success message
-   * @returns notification
-   * @memberof NotificationService
-   */
-  static showSuccessNotification(message) {
-    return NotificationService.showNotification(message, NotificationService.SUCCESS_TYPE, 3000);
-  }
+  return notification;
+}
 
-  /**
-   * Shows the error notification.
-   *
-   * @static
-   * @param {string} message Error message
-   * @returns notification
-   * @memberof NotificationService
-   */
-  static showErrorNotification(message) {
-    return NotificationService.showNotification(message, NotificationService.ERROR_TYPE);
-  }
+/**
+ * Shows the success notification.
+ *
+ * @static
+ * @param {string} message Success message
+ * @returns notification
+ */
+export function showSuccessNotification(message) {
+  return showNotification(message, SUCCESS_TYPE, 3000);
+}
 
-  /**
-   * Shows the warning notification.
-   *
-   * @static
-   * @param {string} message Warning message
-   * @returns notification
-   * @memberof NotificationService
-   */
-  static showWarningNotification(message) {
-    return NotificationService.showNotification(message, NotificationService.WARNING_TYPE);
-  }
+/**
+ * Shows the error notification.
+ *
+ * @static
+ * @param {string} message Error message
+ * @returns notification
+ */
+export function showErrorNotification(message) {
+  return showNotification(message, ERROR_TYPE);
+}
 
-  /**
-   * Shows the info notification.
-   *
-   * @static
-   * @param {string} messageInfo message
-   * @returns notification
-   * @memberof NotificationService
-   */
-  static showInfoNotification(message) {
-    return NotificationService.showNotification(message, NotificationService.INFO_TYPE);
-  }
+/**
+ * Shows the warning notification.
+ *
+ * @static
+ * @param {string} message Warning message
+ * @returns notification
+ */
+export function showWarningNotification(message) {
+  return showNotification(message, WARNING_TYPE);
+}
 
-  /**
-   * Shows the default error notification.
-   *
-   * @static
-   * @returns notification
-   * @memberof NotificationService
-   */
-  static showDefaultErrorNotification() {
-    return NotificationService.showErrorNotification('Trenutno nije moguće dohvatiti podatke. Osvježite stranicu da biste pokušali ponovno.');
-  }
+/**
+ * Shows the info notification.
+ *
+ * @static
+ * @param {string} messageInfo message
+ * @returns notification
+ */
+export function showInfoNotification(message) {
+  return showNotification(message, INFO_TYPE);
 }
 
-export default NotificationService;
+/**
+ * Shows the default error notification.
+ *
+ * @static
+ * @returns notification
+ */
+export function showDefaultErrorNotification() {
+  return showErrorNotification('Trenutno nije moguće dohvatiti podatke. Osvježite stranicu da biste pokušali ponovno.');
```

Comment:
Maybe we should translate this.

Reply:
Yeah, you can create issue to translate whole application (check all and translate to en).